com.unity.ml-agents/Runtime/Inference/TensorNames.cs

```diff
@@ -23,12 +23,16 @@ internal static class TensorNames
         public const string MemorySize = "memory_size";
         public const string VersionNumber = "version_number";
         public const string IsContinuousControl = "is_continuous_control";
-        public const string ActionOutputShape = "action_output_shape";
-        public const string ActionOutput = "action";
+        public const string ActionOutputShapeDeprecated = "action_output_shape";
+        public const string ContinuousActionOutputShape = "continuous_action_output_shape";
+        public const string DiscreteActionOutputShape = "discrete_action_output_shape";
+        public const string ActionOutputDeprecated = "action";
+        public const string ContinuousActionOutput = "continuous_actions";
+        public const string DiscreteActionOutput = "discrete_actions";
 
         public static readonly string[] RequiredConstants =
         {
-            VersionNumber, MemorySize, IsContinuousControl, ActionOutputShape
+            VersionNumber, MemorySize
```

Comment:
These were added because if people passed a non-ML-Agents model without the required tensors, they would get obscure null reference errors when we tried to pull out the constants. For example, in 
```
var modelApiVersion = (int)model.GetTensorByName(TensorNames.VersionNumber)[0];
```
Now that now all of them are required, but the logic is more complicated (e.g. need ActionOutputShape OR ContinuousActionOutputShape OR DiscreteActionOutputShape), we should probably remove that block that checks this, and just make sure that the tensor is there before you use it (and raise a friendly message if it's not).